build/Dependencies.props

```diff
@@ -12,6 +12,7 @@
     <SystemDrawingCommonPackageVersion>4.5.0</SystemDrawingCommonPackageVersion>
     <BenchmarkDotNetVersion>0.11.1</BenchmarkDotNetVersion>
     <TensorFlowVersion>1.10.0</TensorFlowVersion>
+    <SystemCollectionsImmutableVersion>1.5.0</SystemCollectionsImmutableVersion>
 
     <MicrosoftCodeAnalysisCSharpVersion>2.9.0</MicrosoftCodeAnalysisCSharpVersion>
```

Comment:
any chance you know why we have this extra line?

Reply:
Because I'm using `System.Collections.Immutable` and intend to use it more going forward, and we seem to have a policy that we put versions to potentially widely usable nugets in this file.

Reply:
Indeed I think that might be a nice way to solve problems we'll have with accessibility, while not falling prey to the efficiency problems of `IReadOnlyCollection/List`.

---
In reply to: [215833124](https://github.com/dotnet/machinelearning/pull/843#discussion_r215833124) [](ancestors = 215833124)